Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -592,6 +593,13 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
             paywallCache = nil
         }
 
+        let virtualCurrencyManager = VirtualCurrencyManager(
```

Comment:
Perhaps it would make sense to add some kind of diagnostics for Virtual Currencies? Of course, it would be a matter of a separate PR and discussing it internally with the team. But just something to have in mind